docs/design/sno-board.md

```diff
@@ -33,13 +36,15 @@ Ability to graceful exit a satellite or the network in the GUI
 		- The “All Satellites” option should only show information about allocated and used storage space/ bandwidth 
 - When I want to know how much of the allocated storage space my SN has used on the network I want a simple graphic on the SNOBoard so that I can consume this information quickly
 - When I need to know how much of the allocated bandwidth my SN has used I want a simple graphic on the SNOBoard so that I can consume this information quickly
-- When I am looking at the bandwidth graphic I want to be able to distinguish how much of the bandwidth has been egress vs ingress so that I can determine if my current internet is suitable. 
+- When I am looking at the bandwidth graphic I want to be able to distinguish how much bandwidth has been egress vs ingress so that I can determine if my internet plan is suitable for the type of traffic my node is getting. 
```

Comment:
This item of the list ends with period (`.`) meanwhile not all of them do. The majority of the list items of this document don't end with period, so I would remove those from the ones which do in order to keep the format consistent.